module/entities/helper.js

```diff
@@ -981,6 +981,14 @@ module.exports = class EntitiesHelper {
       try {
         let ids = [];
 
+        console.log(            {
```

Comment:
@borkarsaish65 we shoud not be using entity module is what my understanding. we should use entity-management service for all entity related functionalities. see if the entity related controller is also required or not

Reply:
this is not used @VISHNUDAS-tunerlabs , this is done only to refer how the querying was done in enitity module before.